src/game/scenes/SettingsMenuScene.cpp

```diff
@@ -7,35 +7,276 @@
 
 #include "SettingsMenuScene.hpp"
 
+#include "logger/Logger.hpp"
 #include "util/util.hpp"
 
+#include "raylib/core/Audio.hpp"
+#include "raylib/core/Window.hpp"
+#include "raylib/core/scoped.hpp"
+
 #include "game/components/Color.hpp"
+#include "game/components/Controlable.hpp"
 #include "game/components/Model.hpp"
 #include "game/components/Position.hpp"
+#include "game/components/Position2D.hpp"
+#include "game/components/Rectangle.hpp"
 #include "game/components/RotationAngle.hpp"
 #include "game/components/RotationAxis.hpp"
 #include "game/components/Scale.hpp"
 #include "game/components/Size.hpp"
+#include "game/components/Size2D.hpp"
+#include "game/components/Textual.hpp"
+#include "game/gui/components/Checkable.hpp"
+#include "game/gui/components/Clickable.hpp"
+#include "game/gui/components/Widget.hpp"
+
+#include "game/systems/DrawSelectedWidget.hpp"
+#include "game/systems/DrawText.hpp"
+#include "game/systems/InputManager.hpp"
 #include "game/systems/Model.hpp"
+#include "game/systems/Rectangle.hpp"
+
+#include "localization/Localization.hpp"
+#include "localization/Resources.hpp"
+
+static void loadGraphicSettings(ecs::World &world)
+{
+    world.addEntity()
+        .with<game::components::Position>(480.f, 100.f)
+        .with<game::components::Textual>(
+            localization::resources::settings::rsSettingsGraphic, 40, raylib::core::Color::RED)
+        .build();
+
+    world.addEntity()
+        .with<game::components::Position>(500.f, 200.f)
+        .with<game::components::Textual>(
+            localization::resources::settings::rsSettingsFullscreen, 20, raylib::core::Color::RED)
+        .with<game::components::Controlable>(game::User::UserId::User1)
+        .with<game::gui::Widget>(game::SettingsMenuScene::FULLSCREEN, game::SettingsMenuScene::VOLUME_MUTE,
+            game::gui::Widget::NullTag, game::gui::Widget::NullTag, game::SettingsMenuScene::RES_2, true)
+        .with<game::gui::Clickable>(
+            [](ecs::Entity _) {
```

Comment:
You may omit the name of parameters in both functions and lambda expressions.
In this case you can safely remove to ` _` and `(void)_` parts.

Can you do that for the other instances in this file please?